### DISCUSSION TITLE
Wsparcie dla ekspandera PCF8575

### DISCUSSION THREAD

**Discussion by paku2020** [lkankowski/arduino-multi-relay2#39] [Q&A]
Witam,

Szukałem informacji w necie, ale mało jest na ten temat.
W związku z tym, że na rynku brakuje ekspanderów mcp23017  to czy sketch obsługuje ekspander PCF8575?
Według dokumentacji obsługuje PCF8574, a to jest prawie to samo co PCF8575.
Więc czy PCF8575 będzie działał ? :)

Pozdrawiam

**Comment by lkankowski** [comment]
Z tego co pamiętam (nie sprawdzałem tego teraz), to biblioteki do PCF8574 i PCF8575 są niemal identyczne.
Proponuję spróbować podmieniając w kodzie bibliotekę PCF8574 na PCF8575. Jak zadziała, to wystarczy zwiększyć ilość pinów.
Chętnie przyjmę na to pull requesta, albo zwyczajną informację co i jak, to rozszerzę kod i dokumentację.

PS.
Powyżej odnoszę się do biblioteki https://github.com/skywodd/pcf8574_arduino_library

**Comment by paku2020** [comment]
`Proponuję spróbować podmieniając w kodzie bibliotekę PCF8574 na PCF8575. Jak zadziała, to wystarczy zwiększyć ilość pinów.`

Ok to kupię w wolnej chwili ten ekspander i przetestuję. 
Nie jestem programistą i nie wiem gdzie w kodzie zwiększyć ilość  pinów.
Dam znać jak już będę coś wiedział więcej

**Reply by lkankowski** [reply]
Obstawiam, że powinno działać od strzała, bo te chipy i liby są niemal identyczne :)
Jakby coś chętnie pomogę w ramach dostępnego czasu.
